kernelci/kbuild.py

```diff
@@ -110,7 +110,12 @@ def __init__(self, node=None, jobname=None, params=None, jsonobj=None):
             self._artifacts = []
             self._current_job = None
             self._config_full = ''
-            self._srctarball = node['artifacts']['tarball']
+            if node.get('artifacts') and 'tarball' in node['artifacts']:
+                self._srctarball = node['artifacts']['tarball']
+            elif node.get('debug') and 'tarball' in node['debug']:
+                self._srctarball = node['debug']['tarball']
```

Comment:
```suggestion
            if node.get('debug') and 'tarball' in node['debug']:
                self._srctarball = node['debug']['tarball']
            elif node.get('artifacts') and 'tarball' in node['artifacts']:
                self._srctarball = node['artifacts']['tarball']
```

Maybe the other way around? So we always prioritize the `debug` parameters if present